UserAvatar/src/main/java/com/infomaniak/core/useravatar/AvatarData.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.useravatar
+
+import android.os.Parcelable
+import androidx.annotation.ColorInt
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class AvatarData(
+    val uri: String = "",
+    val userInitials: String = "",
```

Comment:
Should these two not be nullable?

Reply:
It doesn't change anything except adding a small null check at usage. 
And it doesn't really add any meaning, so I choose not to make them nullable